components/core/src/clp/clp/FileDecompressor.hpp

```diff
@@ -24,6 +24,14 @@ class FileDecompressor {
             std::unordered_map<std::string, std::string>& temp_path_to_final_path
     );
 
+    bool decompress_to_ir(
```

Comment:
Let's add a docstring. I know the other method doesn't have a docstring, but it should.

Reply:
As discussed, will add in future changelist